src/components/Survey/AudioSpeed/AudioSpeed.tsx

```diff
@@ -15,15 +15,15 @@ const {
 } = style;
 
 interface Props {
-  audioSpeed: number;
-  setAudioSpeed: (value: number) => void;
+  audioSpeed: boolean | null;
+  setAudioSpeed: (audioSpeed: boolean) => void;
   handleSetNextPage: () => void;
 }
 
 const AudioSpeed = ({ audioSpeed, setAudioSpeed, handleSetNextPage }: Props) => {
   const [isDisabled, setIsDisabled] = useState<boolean>(!audioSpeed);
 
-  const handleClickaudioSpeedButton = (userAudioSpeed: number) => () => {
+  const handleClickaudioSpeedButton = (userAudioSpeed: boolean) => () => {
```

Comment:
카멜케이스가 제대로 적용이 안되어있네요 ㅠㅅㅠ
```suggestion
  const handleClickAudioSpeedButton = (userAudioSpeed: boolean) => () => {
```